### PR TITLE
Make sure salt master container is running before declaring success

### PR DESCRIPTION
**Pull request by sayf-eddine-scality**
Fixes: #656
Salt master pod may take some time to start due to image pulling etc, the state deploying it should wait until it is running before declaring success

Fixes: #656

### CODE REVIEW

**Comment by NicolasT** [comment]
Doesn't this mean the `sleep` can be removed as well?

**Comment by sayf-eddine-scality** [comment]
> Doesn't this mean the `sleep` can be removed as well?

Done

File: salt/metalk8s/salt/master/deployed.sls [hunk 1]
```diff
@@ -31,3 +31,12 @@ Install and start salt master manifest:
       salt_master_version: {{ salt_master_version }}
     - require:
       - file: Create salt master directories
+
+Make sure salt master container is up:
```
Comment (alexandre-allard) [inline salt/metalk8s/salt/master/deployed.sls:35]: Shouldn't this command require "Install and start salt master manifest" ?

Comment (sayf-eddine-scality) [reply salt/metalk8s/salt/master/deployed.sls:35]: right

**Comment by sayf-eddine-scality** [comment]
@bert-e approve

**Review by TeddyAndrieux — APPROVED** [review]

**Pull request merged**